libutils/Makefile.am

```diff
@@ -47,6 +47,8 @@ libutils_la_SOURCES = \
 	proc_keyvalue.c proc_keyvalue.h \
 	bool.h \
 	json.c json.h json-priv.h \
+	csv-json.c csv-json.h \
+	env-json.c env-json.h \
```

Comment:
Creating a separate function for each file is a bit excessive. Either add them to `json.[ch]` or create something like `json_utils.[ch]` for all JSON utility functions.